Application/src/view/visuals/component/Message.java

```diff
@@ -231,11 +237,23 @@ public void setStatic(boolean staticIndicator)  {
     public void renderDefault(GraphicsContext gc) {
 
 
+        if(trails.size() != 0) {
+            Trail tempTrail;
+            //Draws all trails except the first and last one
+            for (int i = 1; i < trails.size() - 1; i++) {
+                tempTrail = trails.get(i);
+                gc.drawImage(cloud, tempTrail.getXcoordinate(), (tempTrail.getYcoordinate() + 18), tempTrail.getWidth(), tempTrail.getHeight());
+            }
 
-        //Draw trail for the message, for each instance in the arraylist
-        for (Trail t: trails)
-            gc.drawImage(trail, t.getXcoordinate(), (t.getYcoordinate() + 18), t.getWidth(), t.getHeight());
-
+            Trail last = trails.get(trails.size() - 1);
+            //Puts an arrow on the last location of the trail array
+            //The if statement changes the direction of the arrow if necessary
+            if (rotated) {
```

Comment:
Why is it called rotated? very confusing as to what it means. rotated how? left? right? downwards? upwards? North-west?